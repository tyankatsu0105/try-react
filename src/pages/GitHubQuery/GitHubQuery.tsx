import React, { FunctionComponent } from 'react';
import { Helmet } from 'react-helmet';

import { QueryForm } from './containers';

const GitHubQuery: FunctionComponent = () => {
  return (
    <>
      <Helmet>
        <title>This is GitHubQuery</title>
      </Helmet>

      <QueryForm />
    </>
  );
};

export default GitHubQuery;
