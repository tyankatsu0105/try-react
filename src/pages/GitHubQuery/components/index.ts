export { QueryForm as QueryFormComponent } from './QueryForm';
