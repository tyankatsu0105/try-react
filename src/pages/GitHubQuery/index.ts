import { lazy } from 'react';

export const GitHubQuery = lazy(() => import('./GitHubQuery'));
