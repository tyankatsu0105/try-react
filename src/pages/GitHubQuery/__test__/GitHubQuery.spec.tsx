import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';

import GitHubQuery from '../GitHubQuery';

describe('GitHubQuery', () => {
  it('snapshot', () => {
    const { baseElement } = render(
      <MemoryRouter initialEntries={['/']}>
        <GitHubQuery />
      </MemoryRouter>
    );
    expect(baseElement).toMatchSnapshot();
  });
});
