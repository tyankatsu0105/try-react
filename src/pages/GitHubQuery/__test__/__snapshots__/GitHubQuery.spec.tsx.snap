// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubQuery snapshot 1`] = `
<body>
  <div />
</body>
`;
